app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,212 @@
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@DisplayName("TaskService 클래스")
+public class TaskServiceTest {
+    private static final Long TASK_ID_1 = 1L;
+    private static final Long TASK_ID_2 = 2L;
+    private static final String TASK_TITLE_1 = "task1";
+    private static final String TASK_TITLE_2 = "task2";
+    private static final String NEW_TITLE = "new title";
+
+    private TaskService taskService;
+    private Task task1;
+    private Task task2;
+
+    @BeforeEach
+    void setUp() {
+        taskService = new TaskService();
+        task1 = new Task();
+        task1.setId(TASK_ID_1);
+        task1.setTitle(TASK_TITLE_1);
+        task2 = new Task();
+        task2.setId(TASK_ID_2);
+        task2.setTitle(TASK_TITLE_2);
+    }
+
+    TaskService taskExistService() {
+        taskService.createTask(task1);
+        taskService.createTask(task2);
+        return taskService;
+    }
+
+    Task newTask() {
+        Task task = new Task();
+        task.setId(TASK_ID_1);
+        task.setTitle(NEW_TITLE);
+        return task;
+    }
+
+    @Nested
+    @DisplayName("getTasks 메서드는")
+    class Describe_getTasks {
+        @Nested
+        @DisplayName("할 일 목록이 있다면")
+        class Context_with_tasks {
+
+            @BeforeEach
+            void setUp() {
+                taskService.createTask(task1);
+                taskService.createTask(task2);
+            }
+
+            @DisplayName("할 일 목록을 리턴한다")
+            @Test
+            void it_returns_tasks() {
+                List<Task> tasks = taskService.getTasks();
+
+                assertAll(
+                        () -> assertThat(tasks).hasSize(2),
+                        () -> assertThat(tasks).containsExactly(task1, task2)
+                );
+            }
+        }
+
+        @Nested
+        @DisplayName("할 일인 목록이 없다면")
+        class Context_without_tasks {
+            @DisplayName("비어있는 값을 리턴한다")
+            @Test
+            void it_returns_empty_tasks() {
+                List<Task> tasks = taskService.getTasks();
+
+                assertThat(tasks).isEmpty();
+            }
+        }
+    }
+
+    @Nested
+    @DisplayName("getTask 메서드는")
+    class Describe_getTask {
+        final Long id = 1L;
+
+        @Nested
+        @DisplayName("찾고자하는 할 일이 있으면")
+        class Context_with_task {
+            @BeforeEach
+            void setUp() {
+                taskService.createTask(task1);
+                taskService.createTask(task2);
+            }
+
+            @DisplayName("할 일을 리턴한다")
+            @Test
+            void it_returns_tasks() {
+                assertAll(
+                        () -> assertThat(taskService.getTask(id)).isNotNull(),
+                        () -> assertThat(taskService.getTask(id)).isEqualTo(task1)
+                );
```

Comment:
`isEqualTo`로 같은지 확인하고 있으니 null 인지는 확인하지 않아도 될 것 같아요


```suggestion
                assertThat(taskService.getTask(id)).isEqualTo(task1)
```

Reply:
불필요하게 검증하고 있었던 것같네요